src/views/motor.py

```
@@ -1,73 +1,79 @@
+    propellant_I_12: Optional[Any] = None
+    propellant_I_13: Optional[Any] = None
+    propellant_I_22: Optional[Any] = None
+    propellant_I_23: Optional[Any] = None
+    propellant_I_33: Optional[Any] = None
+    propellant_mass: Optional[Any] = None
+    reshape_thrust_curve: Optional[Any] = None
+    total_mass: Optional[Any] = None
+    total_mass_flow_rate: Optional[Any] = None
+    thrust: Optional[Any] = None
+
+    def __init__(self, **data):
```

Pylint (3.3.7)

[warning] Useless parent or super() delegation in method '__init__'

(W0246)